src/core/models/py_ftrl.cc

```diff
@@ -69,58 +69,71 @@ mantissa_nbits=10, nepochs=1, double_precision=False, negative_class=False,
 interactions=None, model_type='auto', params=None)
 --
 
-Create a new :class:`FTRL <datatable.models.Ftrl>` model object.
+Create a new :class:`Ftrl <datatable.models.Ftrl>` object.
 
 Parameters
 ----------
-alpha: float
-    :math:`\alpha` in per-coordinate learning rate formula, should be
-    positive. See :attr:`.alpha <models.Ftrl.alpha>` for details.
+alpha: float (optional)
```

Comment:
I'm not sure we need these "(optional)" marks. Any parameter that has a default value in the signature is going to be optional.

Reply:
Yes, that's true. I just saw we had it somewhere in the docs and thought we're going to have it everywhere... though it seems now that it was only `kfold` that has that mark.